keps/sig-storage/3756-volume-reconstruction/kep.yaml

```diff
@@ -0,0 +1,47 @@
+title: Robust VolumeManager reconstruction after kubelet restart
+kep-number: 3756
+authors:
+  - "@jsafrane"
+owning-sig: sig-storage
+participating-sigs:
+  - sig-node
+status: provisional
```

Comment:
Are there any outstanding items that need to be resolved before marking this as implementable?

Reply:
No, unless there are review remarks. Marked as implementable.